### PR TITLE
fixed 2 things

### PR DESCRIPTION
**Pull request by binnichtaktiv**

### CODE REVIEW

**Comment by binnichtaktiv** [comment]
nvm it should work im lost as always

File: install-pyzule-gen.sh [hunk 1]
```diff
@@ -1,18 +1,32 @@
 #!/bin/bash
 PZ_DIR=${HOME}/.config/pyzule
 
-if [ ! -d "${PZ_DIR}" ] || [ ! -x "$(command -v pyzule)" ]; then
-    echo "[!] pyzule is not installed"
-    exit 1
+if [ ! -d "${PZ_DIR}" ]; then
+    echo "[*] ${PZ_DIR} does not exist. Creating..."
```
Comment (asdfzxcvbn) [inline install-pyzule-gen.sh:5]: be consistent with style, everything shown to the user is shown in **lowercase** unless it's something case-sensitive

File: install-pyzule-gen.sh [hunk 2]
```diff
@@ -1,18 +1,32 @@
...
+    mkdir -p "${PZ_DIR}"
 fi
 
-echo "[*] installing pyzule-gen.."
+if [ ! -d "${PZ_DIR}/venv" ]; then
+    echo "[*] Creating virtual environment..."
+    python3 -m venv "${PZ_DIR}/venv"
```
Comment (asdfzxcvbn) [inline install-pyzule-gen.sh:11]: don't hardcode `python3`, use something like https://github.com/asdfzxcvbn/pyzule/blob/2e4e024b3063c161194a581bfdd5354faf338433/install-pyzule.sh#L16 instead

File: install-pyzule-gen.sh [hunk 3]
```diff
@@ -1,18 +1,32 @@
...
-    sudo sed -e "1s|.*|#\!${PZ_DIR}/venv/bin/python|" -i "" /usr/local/bin/pyzule-gen  # bsd sed is broken asf
+    sudo sed -e "1s|.*|#\!${PZ_DIR}/venv/bin/python|" -i "" /usr/local/bin/pyzule-gen
 fi
-echo "[*] fixed interpreter path!"
+
+echo "[*] Fixed interpreter path!"
 sudo chmod +x /usr/local/bin/pyzule-gen
-echo "[*] done!"
+
+echo "[*] pyzule-gen is ready to use!"
```
Comment (asdfzxcvbn) [inline install-pyzule-gen.sh:31]: unnecessary comment

File: pyzule-gen-ish.py [hunk 4]
```diff
@@ -1,113 +1,113 @@
...
+    if real_args[key]:
+        real_args[key] = True
+    else:
+        del real_args[key]
+used = {k: v for k, v in real_args.items() if v}
+
+print("[*] generating..")
+with ZipFile(args.o, "w", ZIP_DEFLATED, compresslevel=4) as zf:
+    with zf.open("config.json", "w") as f:
+        f.write(json.dumps(used).encode('utf-8'))
```
Comment (asdfzxcvbn) [inline pyzule-gen-ish.py:97]: be consistent with the rest of the quotes, switch the single quotes to double quotes

**Review by asdfzxcvbn — COMMENTED** [review]
pls

**Review by asdfzxcvbn — CHANGES_REQUESTED** [review]
needs

**Review by binnichtaktiv — COMMENTED** [review]
w

**Pull request closed without merging**